src/clientStub.ts

```diff
@@ -160,7 +160,8 @@ export class DgraphClientStub {
 
 function ensureCallOptions(options?: grpc.CallOptions): grpc.CallOptions {
     return (options === undefined) ? {
-        propagate_flags: grpc.propagate.DEFAULTS,
+        // @ts-ignore
+        propagate_flags: grpc.propagate ? grpc.propagate.DEFAULTS : 4,
```

Comment:
What is this magic number 4?

Reply:
I don't know this flag how to work and grpc-js module don't have implemented this flag, so I set the value same as grpc module `grpc.propagate.DEFAULTS` value `4`